fix.sh

```diff
@@ -0,0 +1,7 @@
+# fixes OpenSSL webpack error
+# It changes MD4 to SHA256
+# I "stole" the code from
+# https://stackoverflow.com/questions/69394632/webpack-build-failing-with-err-ossl-evp-unsupported
+
+cp ./fixes/webpack.fix ./node_modules/@nuxt/webpack/dist/webpack.js &&
+cp ./fixes/custom.fix ./node_modules/@types/custom.d.ts
```

Comment:
I just tested on a cleaned repo, and it doesn't seem like we need these anymore. 

If we do, we might want to use [patch-package](https://www.npmjs.com/package/patch-package) instead.